samples/dashboards/edit-mode/change-close-icon/demo.details

```diff
@@ -0,0 +1,6 @@
+---
+ name: Change close icon
```

Comment:
Do we have any convention for the naming here?

Reply:
Not that I am aware of. I notice this text only in the sidepanel of JSfiddle.